src/main/java/com/checkmarx/flow/config/FlowConfig.java

```diff
@@ -66,6 +68,26 @@ public JavaMailSender getJavaMailSender() {
         return mailSender;
     }
 
+    public TemplateEngine getTemplateEngine() {
+
+        TemplateEngine templateEngine = new TemplateEngine();
+        templateEngine.setTemplateResolver(fileTemplateResolver());
+
+        return templateEngine;
+
+    }
+
+    public FileTemplateResolver fileTemplateResolver() {
```

Comment:
1) It's better if method name contains an action (e.g. getFileTemplateResolver).
2) Can it be private?

Reply:
Done!